purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -1302,10 +1298,10 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             }
     }
 
-    private val handler: Handler? = Handler(Looper.getMainLooper())
+    private val handler = Handler(Looper.getMainLooper())
```

Comment:
Honestly, not sure why that was marked as nullable... It was always non null, lmk if I missed anything.